docs/form-app-script.js

```diff
@@ -0,0 +1,120 @@
+// This is the code behind the Google Form that is the first-round version of campaign creation: https://goo.gl/forms/aAcBrU408Kv2jrEr2
+
+// TODO change these to prod URLs eventually
+SPONSOR_URL = 'http://jingle.staging.prx.tech/api/v1/sponsors';
```

Comment:
Would it be possible to make these configurable?  Maybe some sort of hidden form-field, pre-populated with `jingle.staging.prx.tech`?

Reply:
Though as I'm googling for that, it may be impossible.

Reply:
hmm, i'm not sure. will hunt around the docs!